lib/features/thread/presentation/thread_view.dart

```diff
@@ -864,11 +866,14 @@ class ThreadView extends GetWidget<ThreadController>
 
   Widget _buildMarkAsMailboxReadLoading(BuildContext context) {
```

Comment:
Also should rename it 

```suggestion
  Widget _buildMailboxActionProgressBanner(BuildContext context) {
```

Reply:
done